src/pydata_sphinx_theme/assets/styles/sections/_container.scss

```diff
@@ -1,10 +1,16 @@
 .bd-container {
   flex-grow: 1;
   display: flex;
-  flex-direction: column;
+  justify-content: center;
 
   .bd-container__inner {
-    flex-grow: 1;
-    justify-content: center;
+    display: flex;
+  }
+}
+
+.bd-page-width {
+  width: 100%;
+  @include media-breakpoint-up(lg) {
+    width: $breakpoint-page-width;
```

Comment:
why not just set `max-width: $breakpoint-page-width` instead of a fixed width?

Reply:
True, I will use max-width. I was following the `sphinx-basic-ng` convention https://github.com/pradyunsg/sphinx-basic-ng/blob/main/src/sphinx_basic_ng/theme/basic-ng/static/skeleton.css#L294 , but it has different fixed widths for a lot of sizes.

Reply:
If you think it's better to fix the width, either for our own reasons or just to be the same as the basic-ng theme, I think it's fine. We just need to encode this context in the CSS comments or variable names so it doesn't seem like a random CSS decision

Reply:
I have added a comment at the location where the variable is declared https://github.com/pydata/pydata-sphinx-theme/pull/861/files#diff-e1eb68fda05547f5b24fb1a5b0a020546ba808616976abb40a6b704bddea3858R23